### DISCUSSION TITLE
Run decompile with response in CMD

### DISCUSSION THREAD

**Discussion by StepanGavrilov** [NationalSecurityAgency/ghidra#4152] [Q&A]
I decompiled a base file:

![Снимок 13 04 2022 в 18 40](https://user-images.githubusercontent.com/55409307/163226998-e07d83a1-33eb-49a1-8b4b-0a4d719a6631.jpg)
![Снимок 13 04 2022 в 18 39](https://user-images.githubusercontent.com/55409307/163227003-b0700866-4234-4df6-a0a6-6363f01a6376.jpg)

```
#include <stdlib.h>

int main() {
    void *p = malloc(200);
    free(p);
    free(p);
    return 0;
}
```

How i get response of decompiling like in screenshots, but with using ghidra with cmd commands or any utilities ?

**Comment by ghidracadabra** [comment]
The script `ShowCCallsScript.java` (included with Ghidra) should get you most of the way there.  Just add the line
`printf("%s\n", decompRes.getDecompiledFunction().getC());` 
to the method `decompileFunction` after the definition of `decompRes`
